### ISSUE TITLE
SPDX documentation not in sync with schema

### ISSUE THREAD

**Issue by Moullisha** [spdx/spdx-spec#872]
documentNamespace is a required field in v2.3.0 as per documentation but has not been mentioned in the required array in schema

![image](https://github.com/spdx/spdx-spec/assets/65083195/08d92f1d-51b7-46a3-9465-c6cf1258c11d)

**Comment by goneall** [comment]
It looks like it is included in the required array here:

https://github.com/spdx/spdx-spec/blob/8a595028b0386138c84f7188237787c374f6a6cf/schemas/spdx-schema.json#L748

**Comment by Moullisha** [comment]
Yes, it is mentioned as a required field in [development/v2.3.1](https://github.com/spdx/spdx-spec/blob/development/v2.3.1/schemas/spdx-schema.json) but not in [development/v2.3](https://github.com/spdx/spdx-spec/blob/development/v2.3/schemas/spdx-schema.json) which created confusion. Do you suggest we follow the 2.3.1 branch?

**Comment by goneall** [comment]
> Yes, it is mentioned as a required field in [development/v2.3.1](https://github.com/spdx/spdx-spec/blob/development/v2.3.1/schemas/spdx-schema.json) but not in [development/v2.3](https://github.com/spdx/spdx-spec/blob/development/v2.3/schemas/spdx-schema.json) which created confusion.

Good point.  This must have been an issue identified in 2.3.0 and fixed in the development branch.

> Do you suggest we follow the 2.3.1 branch?

2.3.1 is not yet released, so there is some risk of picking up a change that may be amended later.  That being said, we are only fixing critical issues in 2.3 at the moment, so you could follow the 2.3.1 branch if you want the most up to date fixes.

If the tech workgroup had more bandwidth, we'd probably do a patch release of 2.3.1, but we're all working hard to get the 3.0 release out.

**Comment by bact** [comment]
Btw, the in-development version of v2.3.1 is now published as "v2.3.1-dev"

See for example here:
https://spdx.github.io/spdx-spec/v2.3.1-dev/document-creation-information/#65-spdx-document-namespace-field 

The schema browser is here:
https://spdx.github.io/spdx-spec/v2.3.1-dev/spdx-json-schema.html

**Comment by goneall** [comment]
Closing this as resolved in 2.3.1 - BTW - we're working on a release for 2.3.1 - stay tuned
